app/components/publications/publication-sidebar.hbs

```diff
@@ -142,15 +142,9 @@
           <div class="auk-form-group">
             <WebComponents::AuLabel>{{t 'final-publication-date'}}</WebComponents::AuLabel>
             <WebComponents::AuDatepicker
-                    @defaultDate={{this.publicationFlow.publishBefore}}
+                    @defaultDate={{this.publicationSubcase.dueDate}}
                     @enable={{this.allowedUltimatePublicationDates}}
                     @onChange={{this.setUltimatePublicationDate}}/>
-            {{#if this.publicationFlow.publicationBeforeDateHasExpired}}
-              <div class="auk-form-help-text auk-form-help-text--warning">
-                <WebComponents::AuIcon @name="alert-triangle"/>
-                {{t "date-expired"}}
-              </div>
-            {{/if}}
```

Comment:
Deze verwijdering.
valt dit onder de noemer "Validatielogica rond datums is verwijderd"?
Willen we in de sidebar niet meer tonen als een datum is verstreken? (zou kunnen, ben niet mee met alle designs)